src/routes/(site)/blog/[id]/+page.svelte

```diff
@@ -5,8 +5,24 @@
   import { Temporal } from '@js-temporal/polyfill';
   import Labels from '$lib/components/blog/labels.svelte';
   import BlogBody from './blog-body.svelte';
-
+  import { onMount } from 'svelte';
+  import { toast, ToastType } from '$lib/components/toaster/toasts';
   export let data: PageData;
+  onMount(() => {
+    const copyBtns = document.querySelectorAll('.copy-code');
+    for (let i = 0; i < copyBtns.length; i++) {
+      copyBtns[i].addEventListener('click', (event: any) => {
+        console.log(event.target);
+        let content = event.target.offsetParent.attributes.getNamedItem(
```

Comment:
move this to different file and call instead https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/src/routes/(site)/events/event.svelte#L17

Reply:
also prefer const instead of let here